### PR TITLE
Add release highlights for 7.6

### PR DESCRIPTION
**Pull request by polyfractal**
@mayya-sharipova @iverase Your PRs (optimize sorts on longs, histograms) were marked as `release-highlight` so I put together a little summary for each.  Let me know if I described it incorrectly or missed something important.

### CODE REVIEW

File: docs/reference/release-notes/highlights-7.6.0.asciidoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+++++
+<titleabbrev>7.6.0</titleabbrev>
+++++
+
+//NOTE: The notable-highlights tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-highlights[]
+[float]
+==== Histogram field type
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights-7.6.0.asciidoc]:
```suggestion
==== New histogram field type
```

File: docs/reference/release-notes/highlights-7.6.0.asciidoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+A new {ref}/histogram.html[histogram field type] has been added.  The new `histogram` field accepts
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights-7.6.0.asciidoc]:
```suggestion
A new {ref}/histogram.html[histogram field type] has been added. The new `histogram` field accepts
```

File: docs/reference/release-notes/highlights-7.6.0.asciidoc [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+pre-aggregated histograms which can later be used directly in the
+{ref}/search-aggregations-metrics-percentile-aggregation.html[percentiles] and
+{ref}/search-aggregations-metrics-percentile-rank-aggregation.html[percentile_ranks] aggregations.
+This allows users to pre-aggregate histogram data locally to the client and only send the final
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights-7.6.0.asciidoc]:
```suggestion
This allows users to pre-aggregate histogram data locally and only send the final
```

File: docs/reference/release-notes/highlights-7.6.0.asciidoc [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+datastructure, saving storage and network transfer, but still retaining the ability to
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights-7.6.0.asciidoc]:
```suggestion
data structure, saving storage and network bandwidth while retaining the ability to
```

File: docs/reference/release-notes/highlights-7.6.0.asciidoc [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+aggregate it like any other data.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Optimize sorting on long field types
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights-7.6.0.asciidoc]:
```suggestion
==== Optimized sorting on long field types
```

File: docs/reference/release-notes/highlights-7.6.0.asciidoc [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+Sorting on a `long` field now internally rewrites into a Lucene `DistanceFeatureQuery`.
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights-7.6.0.asciidoc]:
```suggestion
Sorting on a {ref}/number.html[`long`] field now internally rewrites into a Lucene `DistanceFeatureQuery`.
```

File: docs/reference/release-notes/highlights-7.6.0.asciidoc [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+This allows skipping non-competitive hits which leads to increased query performance.
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights-7.6.0.asciidoc]:
```suggestion
This lets {es} skip non-competitive hits, which often improves query speed.
```

File: docs/reference/release-notes/highlights-7.6.0.asciidoc [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+In benchmarks this has sped up sorts on `long` fields by 10x.
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights-7.6.0.asciidoc]:
```suggestion
In benchmarking tests, this sped up sorts on `long` fields by 10x.
```

**Review by jrodewig — APPROVED** [review]
Left a few minor editorial nits, but nothing blocking. Thanks @polyfractal.

**Comment by polyfractal** [comment]
Thanks @jrodewig!

**Pull request merged**